bazel/external/requirements.in

```diff
@@ -0,0 +1,27 @@
+# requirements.in
+setuptools
+grpcio-tools
+grpcio
```

Comment:
Should we start a README in this directory with information on how the requirements.in/txt files are to be updated?

Reply:
Right, that would support the developers in adding missing dependencies.

Reply:
I inserted an initial version of the README file with information about the requirements update process.